api/genomics/src/test/java/org/pmiops/workbench/genomics/RandomizeVcfTest.java

```diff
@@ -0,0 +1,63 @@
+package org.pmiops.workbench.genomics;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFileReader;
+import java.io.File;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RandomizeVcfTest {
+  private static final VCFFileReader reader =
+      new VCFFileReader(new File("src/test/resources/NA12878_204126160130_R01C01.toy.vcf.gz"));
+  private static final VariantContext variantContext = reader.iterator().next();
+  private static final RandomizeVcf randomizeVcf = new RandomizeVcf();
+
+  @BeforeClass
+  public static void setUp() {
+    randomizeVcf.sampleNameSuffix = "0";
```

Comment:
nit: I'd probably just expose two constructors here, 1 default constructor, 1 that takes the sampleNameSuffix, used by the test (and `@VisibleForTesting`)

Reply:
oh that would make more sense wouldn't it